From 2356444f7deccf0a5af20f54d3da82fc819f2981 Mon Sep 17 00:00:00 2001
From: "Eric S. Raymond" <esr@thyrsus.com>
Date: Sat, 3 Dec 2016 10:10:27 -0500
Subject: [PATCH 151/268] Address Gitlab issue #187: Bug: ntpq -c ":config" not
 accepted.

It's not worth the additional code complexity to "fix" this.
---
 docs/includes/ntpq-body.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/docs/includes/ntpq-body.txt b/docs/includes/ntpq-body.txt
index a4ab055..0523a99 100644
--- a/docs/includes/ntpq-body.txt
+++ b/docs/includes/ntpq-body.txt
@@ -565,4 +565,6 @@ by 'name', a shortname for the driver type.
 
 The -O (--old-rv) option of legacy versions has been retired.
 
+The command ":config" is no longer accepted; use "config" instead.
+
 // end
-- 
2.7.4

